### DISCUSSION TITLE
Welcome to prayer-times-extension Discussions!

### DISCUSSION THREAD

**Discussion by mohamedmansour** [mohamedmansour/prayer-times-extension#31] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tarekshalaby21** [comment]
السلام عليكم 
thank you so much for this extension and actually it's very important, but there are some issues with it with the new updates with chrome-like I can't choose my location which makes all times wrong, so please fix this issue and make it handier like adding reminder before and after azan and asking Allah to help you to make it better and better and help Muslims everywhere 

,جزاكم الله خيرا
